gensrc/thrift/InternalService.thrift

```diff
@@ -405,7 +405,7 @@ struct TQueryGlobals {
 
   31: optional i64 timestamp_us
 
-  32: optional i64 scan_node_number
+  32: optional i64 connector_scan_node_number
```

Comment:
so there maybe some unexpected but acceptable behavior?

Reply:
no. this variable is introduced by and use by me only.